config/src/pipeline/mod.rs

```diff
@@ -0,0 +1,21 @@
+use serde::{Serialize, Deserialize};
+pub mod station;
+pub mod station_config;
+pub mod events;
+pub mod metadata;
+pub mod step;
+pub mod step_config;
+
+use station::Station;
+use events::PipelineEvent;
+use metadata::Metadata;
+
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct Pipeline {
+    #[serde(flatten)]
+    pub metadata: Metadata,
+    /// stations in the pipeline.
+    pub stations: Vec<Station>,
+    // pipeline events to run this pipeline
+    //pub events: Vec<PipelineEvent>,
```

Comment:
could his be added back in with the default serde attribute so it's no forgotten... I'm already asking you to alter this PR due to that unit test with the panic in it

Reply:
We have updated this. We do not use default because I think there needs to be an event. but Pipeline events actually work now try using the example yaml as it is tested and should be working.